internal/venice-common/src/main/java/com/linkedin/venice/ConfigKeys.java

```diff
@@ -376,6 +376,11 @@ private ConfigKeys() {
   public static final String KME_REGISTRATION_FROM_MESSAGE_HEADER_ENABLED =
       "kme.registration.from.message.header.enabled";
 
+  /**
+   * The following config is to control whether to enable backup version cleanup based on retention policy or not at cluster level.
```

Comment:
this comment doesn't seem to be accurate. Isn't this config for enabling the OFFLINE replica?